src/utils/IDMassAccuracy.cpp

```diff
@@ -382,7 +375,14 @@ class TOPPIDMassAccuracy :
             gpl_out << "set xlabel \"error in Da\"" << endl;
           }
           gpl_out << "set ylabel \"frequency\"" << endl;
-          gpl_out << "plot '" << precursor_out_file << "_gnuplot.dat' title 'Precursor mass error distribution' w boxes, f(x) w lp title 'Gaussian fit of the error distribution'" << endl;
+          gpl_out << "plot '-' using 1:2 title 'Precursor mass error distribution' w boxes, f(x) w lp title 'Gaussian fit of the error distribution'" << endl;
+
+	  for (vector<DPosition<2> >::const_iterator it = values.begin(); it != values.end(); ++it)
+          {
+            out << it->getX() << " " << it->getY() << endl;
```

Comment:
`endl` is really bad for write speed. If you touch this, 100 charma points for switching to `'\n'`